modules/wallet/defrag.go

```diff
@@ -15,6 +15,7 @@ var (
 // managedCreateDefragTransaction creates a transaction that spends multiple existing
 // wallet outputs into a single new address.
 func (w *Wallet) managedCreateDefragTransaction() ([]types.Transaction, error) {
+	dustThreshold := w.managedDustThreshold()
 	minFee, _ := w.tpool.FeeEstimation()
```

Comment:
same here, probably worth indicating that these fields are needed after the lock is acquired, but have to be acquired separate from the lock.

Reply:
You already have the fee here (with `minFee`), you can avoid the additional tpool call and just:

```go
dustThreshold := minFee.Mul64(3)
```

Reply:
@avahowell I did not inline the function because of my other comment and because it seemed more clear. But if it has to be inlined or the extra call is a performance issue I'll change it right away.